x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+interface AdvancedPolicySchemaType {
+  key: string;
+  first_supported_version: string;
+  last_supported_version?: string;
+  documentation: string;
+}
+
+export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
+  {
```

Comment:
@ferullo please review this schema file

Reply:
also, who should be in charge of adding documentation?

Reply:
Do you mean official user documentation? I woudl say lets make Janeene aware of these new advanced options